src/coreclr/inc/crsttypes_generated.h

```diff
@@ -50,92 +50,91 @@ enum CrstType
     CrstException = 32,
     CrstExecutableAllocatorLock = 33,
     CrstExecuteManRangeLock = 34,
```

Comment:
Should the line for the removed item be commented out, so the other locks retain their identifiers? Is there telemetry or tooling that keys off these IDs?

Reply:
Nope. This is used for the specific build only and there is no use beyond internal IDs for critical sections.